### ISSUE TITLE
Browser Integration - Http Auth only Entries

### ISSUE THREAD

**Issue by sidschingis** [keepassxreboot/keepassxc-browser#680]
Is it possible to mark an entry to be only sent for HTTP authentication?

## Expected Behavior
Have two entries on an url, e.g. "sub.domain.com", one for the HTTP Auth, one for the actual user account.
The HTTP Auth dialog automatically takes the HTTP Auth credentials.
Any Login forms on the page do not show the HTTP Auth entry.

## Current Behavior
Login Forms and HTTP Auth dialog show both entries.

## Possible Solution
Add a "HTTP Auth only" checkbox in the Browser Integration settings for the entry

KeePassXC - 2.5
KeePassXC-Browser -1 .5.3
Operating system: Win
Browser: Firefox

**Comment by varjolintu** [comment]
This will be done to KeePassXC. Probably the settings is called "Use this entry only for HTTP Basic Authentication".

**Comment by varjolintu** [comment]
Fixed with: https://github.com/keepassxreboot/keepassxc/pull/3927

**Comment by RolandSM** [comment]
Hello,

I found this option and selected, I have one HTTP auth for a Domain and a seconde for the admin login into the cms.

Every http auth is not working for me not on firefox and not on chrome. I have enabled in the browser plugin to fill and send on http. The popup comes of the request of all entries and I can allow or deny. next time the popup is only coming again but filles not the form...

Greets
